faketouches.js

```diff
@@ -16,6 +16,7 @@
         this.touches = [];
         this.touch_type = FakeTouches.TOUCH_EVENTS;
         this.has_multitouch = true;
+        this.use_prefixed_pointer_event_names = true;
```

Comment:
Its a valid approach (if it works), but for my use case I needed to run a test against both cases, with and without prefixes. The configuration option was the easiest way to do so. Just a thought:

```javascript
this.use_prefixed_pointer_event_names = 'onpointerdown' in document.body;
```

That way, it could still be overridden.

Another though: Modernizr has a very rigorous check for supported events: https://github.com/Modernizr/Modernizr/blob/master/src/hasEvent.js

Might be worthwhile to include modernizr for feature detection. I have a gut feeling that this `eventName in document.body` pattern yields a lot of false-negatives.